cinema/serializers.py

```diff
@@ -78,3 +80,31 @@
     class Meta:
         model = MovieSession
         fields = ("id", "show_time", "movie", "cinema_hall")
+
+
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("id", "movie_session","row", "seat")
+
+    def validate(self, attrs):
+        Ticket.validate_data(
+            attrs["seat"],
+            attrs["row"],
+            attrs["movie_session"].cinema_hall.seats_in_row,
+            attrs["movie_session"].cinema_hall.rows,
+            serializers.ValidationError
+        )
+
+class OrderSerializer(serializers.ModelSerializer):
+    tickets = TicketSerializer(many=True, read_only=False)
+    class Meta:
+        model = Order
+        fields = ("id", "created_at", "tickets")
+    with transaction.atomic():
```

Comment:
The `with transaction.atomic():` block is incorrectly used here. The `create` method should not be indented inside this block. Instead, the transaction should be managed within the method itself.